CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
    + Allows using or across related tables conditions
  - #1100, Customizable OpenAPI title - @AnthonyFisi
 
+### Fixed
+
+ - #2648, Fix inaccurate error codes with new ones - @laurenceisla
+   + `PGRST204`: Column is not found
```

Comment:
My bad. Since this entry is related to https://github.com/PostgREST/postgrest/pull/2542, and this one is not yet released.. then it doesn't qualify as a "fix".

Reply:
Ah, it was already released in [10.1.1](https://github.com/PostgREST/postgrest/blob/main/CHANGELOG.md#1011---2022-11-08)

Reply:
Right, my bad. I forgot about that.